ci/patch_src_in_place

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # shellcheck disable=SC2046,SC2035
-codespell -w --ignore-words-list dedup,nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled,controllerd --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden $(git ls-tree --full-tree --name-only HEAD)
+codespell -w --ignore-words-list dedup,nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled,controllerd,gae,fpr,convience --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden $(git ls-tree --full-tree --name-only HEAD)
```

Comment:
Should we ignore `convience `? Seems like a tyop...

Reply:
Ah, hmm. But you're skipping `src/control/vendor/*`, right?

Reply:
Good point.  I can't find a build which has failed now, if you can point me at one I'll check.